esrally/racecontrol.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, name, description, target, stable=True):
         :param name: A short name of the pipeline. This name will be used to reference it from the command line.
         :param description: A human-readable description what the pipeline does.
         :param target: A function that implements this pipeline
-        :param stable True iff the pipeline is considered production quality.
+        :param stable True if the pipeline is considered production quality.
```

Comment:
For what it's worth, this is probably not a typo, "iff" is a mathematical abbrevation meaning "if and only if". But it does not add anything here, so "if" is fine.